src/components/change-requests/review-change-request/review-change-request.test.tsx

```diff
@@ -3,29 +3,29 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { exampleStandardChangeRequest } from "../../../test-support/test-data/change-requests.stub";
-import { render, screen, routerWrapperBuilder } from "../../../test-support/test-utils";
-import ReviewChangeRequest from "./review-change-request";
+import { exampleStandardChangeRequest } from '../../../test-support/test-data/change-requests.stub';
+import { render, screen, routerWrapperBuilder } from '../../../test-support/test-utils';
+import ReviewChangeRequest from './review-change-request';
 
 const renderComponent = (option: 'Accept' | 'Deny', route: string) => {
-    const RouterWrapper = routerWrapperBuilder({ path: '/change-requests/:id', route });
-    return render(
-        <RouterWrapper>
-            <ReviewChangeRequest option={option} />
-        </RouterWrapper>
-    );
+  const RouterWrapper = routerWrapperBuilder({ path: '/change-requests/:id', route });
```

Comment:
You might consider replacing these hard-coded strings for the proper imported variables while you’re at it here, or just create a ticket for it since I don’t recall there being one.

Reply:
I think it'd be better to leave it as a feature branch. I don't want to have some of the newer devs see that the forms are completely different since the development of this upgrade is probably not something well known among most of our devs.

Reply:
> 

I think it'd be better to leave it as a feature branch. I don't want to have some of the newer devs see that the forms are completely different since the development of this upgrade is probably not something well known among most of our devs.